addons/mail/wizard/mail_wizard_invite.py

```diff
@@ -57,37 +57,16 @@ def add_followers(self):
             document.message_subscribe(partner_ids=new_partners.ids)
 
             model_name = self.env['ir.model']._get(wizard.res_model).display_name
-            # send an email if option checked and if a message exists (do not send void emails)
-            if wizard.send_mail and wizard.message and not wizard.message == '<br>':  # when deleting the message, cleditor keeps a <br>
-                message = self.env['mail.message'].create(
-                    self._prepare_message_values(document, model_name, email_from)
+            # send a notification if option checked and if a message exists (do not send void notifications)
+            if wizard.notify and wizard.message and not is_html_empty(wizard.message):
+                document.message_notify(
+                    body=wizard.message,
+                    email_add_signature=True,
+                    partner_ids=new_partners.ids,
+                    record_name=document.display_name,
+                    reply_to=email_from,
+                    reply_to_force_new=True,
+                    subject=_('Invitation to follow %(document_model)s: %(document_name)s',
+                              document_model=model_name, document_name=document.display_name),
                 )
-                email_partners_data = []
-                recipients_data = self.env['mail.followers']._get_recipient_data(document, 'comment', False, pids=new_partners.ids)[document.id]
-                for _pid, pdata in recipients_data.items():
-                    pdata['notif'] = 'email'
-                    email_partners_data.append(pdata)
-
-                document._notify_thread_by_email(
-                    message, email_partners_data,
-                    send_after_commit=False
-                )
-                # in case of failure, the web client must know the message was
-                # deleted to discard the related failure notification
-                self.env['bus.bus']._sendone(self.env.user.partner_id, 'mail.message/delete', {'message_ids': message.ids})
-                message.unlink()
         return {'type': 'ir.actions.act_window_close'}
-
-    def _prepare_message_values(self, document, model_name, email_from):
```

Comment:
I added a hook in 16 for Yenthe, I vote to keep it :) (would return kwargs to inject into message_notify ).

Reply:
Yes sorry about that, I have re-added and used it.